src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ConfiguredCancelableAsyncEnumerable.cs

```diff
@@ -10,6 +10,9 @@ namespace System.Runtime.CompilerServices
     /// <summary>Provides an awaitable async enumerable that enables cancelable iteration and configured awaits.</summary>
     [StructLayout(LayoutKind.Auto)]
     public readonly struct ConfiguredCancelableAsyncEnumerable<T>
+#if NET9_0_OR_GREATER
```

Comment:
The public API surface of .NET 9 has already been shipped. We are probably not able to ship the update for .NET 9, but only for .NET 10+.

Reply:
Afaik this is not a breaking change. The change actually just expands possible usage, it does not constrain usage. But I can change the if of course if need be...

Reply:
Shipped API surface can't be changed in any form (except trivial bug-fixes). Non-breaking nature only allows it to happen in future version.